templates/user/user_layout.html.twig

```diff
@@ -23,32 +23,26 @@
             </div>
             <div class="pt-5 mt-5 d-flex align-items-stretch user_menu">
                 {% block nav %}
-                    {% if is_granted('ROLE_ADMIN') %}
-                        <a href="{{ path('easyadmin') }}">
-                            <div class="btn btn-info" id ="admin-button">Admin</div>
-                        </a>
-                    {% endif %}
                     {% if 'user_account' != app.request.get('_route') %}
                         <a href="{{ path('user_account') }}">
                             <div class="btn btn-secondary">Back Home</div>
                         </a>
-                        {% if 'submit' not in app.request.get('_route') %}
-                            <a href="{{ path('user_submits') }}">
-                                <div class="btn btn-secondary">Submits</div>
-                            </a>
-                        {% endif %}
-                        {% if 'talk' not in app.request.get('_route') %}
-                            <a href="{{ path('user_talks') }}">
-                                <div class="btn btn-secondary">Talks</div>
-                            </a>
-                        {% endif %}
-                        {% if 'participation' not in app.request.get('_route') %}
-                            <a href="{{ path('user_participations') }}">
-                                <div class="btn btn-secondary">Participations</div>
-                            </a>
-                        {% endif %}
-                        <a href="{{ path('user_account') }}">
-                            <div class="btn btn-secondary">Edit Account</div>
+                        <a href="{{ path('user_submits') }}">
+                            <div class="btn btn-{{ path('user_submits') == app.request.requestUri ? 'primary' : 'secondary' }}">Submits</div>
```

Comment:
fragile too :) 

There is two way to solve that

1. app.request.attribute.get('_route') ==
2. declare a variable `section` (for example) in this template, and override in all subtemplate.

Usually, we do 2, because it allow to highlite a link on many different pages

For example, on 
* https://jolicode.com/blog/
* https://jolicode.com/blog/jai-teste-tailwind-css

=> Two different actions, two different template, but the same menu is highlighted